cmd/minikube/cmd/dashboard.go

```diff
@@ -65,19 +65,30 @@ var dashboardCmd = &cobra.Command{
 		}
 		cluster.EnsureMinikubeRunningOrExit(api, 1)
 
+		fmt.Fprintln(os.Stderr, "Enabling dashboard ...")
+		// Enable the dashboard add-on
+		err = configcmd.Set("dashboard", "true")
```

Comment:
is this idempontent? If yes, then we don't need "Enabling dashboard ..." the second time (I think addons enable is idempotent) - would be nice to check for enablement.